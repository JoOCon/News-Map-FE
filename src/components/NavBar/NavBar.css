.nav-container {
  height: 100vh;
  width: 14rem;
  position: absolute;
  z-index: 10;
  right: 0%;
  display: flex;
  flex-direction: column;
  background: #dadada;
  animation: slideInRight 0.3s linear;
  color: #817b7b; }

.active {
  background-color: #c9c9c9; }

.nav-link {
  text-decoration-line: none;
  font-size: 1.2rem;
  padding: 1rem 0 1rem 1rem;
  color: #6d6d6d; }

.nav-link:hover {
  border-left: 4px solid #797979;
  background-color: #c9c9c9;
  color: #ffffff; }

.greeting {
  margin: 4rem auto 2rem auto; }

@keyframes slideInRight {
  from {
    -webkit-transform: translate3d(100%, 0, 0);
    transform: translate3d(100%, 0, 0);
    visibility: visible; }
  to {
    -webkit-transform: translate3d(0, 0, 0);
    transform: translate3d(0, 0, 0); } }
