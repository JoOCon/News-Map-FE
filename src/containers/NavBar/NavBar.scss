.nav-container {
  height: 100vh;
  width: 14rem;
  position: absolute;
  z-index: 10;
  right: 0%;
  display: flex;
  flex-direction: column;
  background-color: #e2e7ef;

  animation: slideInRight 0.3s linear;
  color: #817b7b;
}

.active {
  background-color: rgb(201, 201, 201);
}

.nav-link {
  text-decoration-line: none;
  font-size: 1.2rem;
  padding: 1rem 0 1rem 1rem;
  color: #6d6d6d;
}

.nav-link:hover {
  border-left: 4px solid rgb(121, 121, 121);
  background-color: rgb(201, 201, 201);
  color: #ffffff;
}

.greeting {
  margin: 4rem auto 2rem auto;
}

@keyframes slideInRight {
  from {
    -webkit-transform: translate3d(100%, 0, 0);
    transform: translate3d(100%, 0, 0);
    visibility: visible;
  }

  to {
    -webkit-transform: translate3d(0, 0, 0);
    transform: translate3d(0, 0, 0);
  }
}
