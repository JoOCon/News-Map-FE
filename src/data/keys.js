export const googleClientId =
  '241779944002-qa42v6cvk3kpu8da0jimdr364vn41t1r.apps.googleusercontent.com';
