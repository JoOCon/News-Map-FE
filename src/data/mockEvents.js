export const eventsResponse = [
  {
    name: 'P!NK: BEAUTIFUL TRAUMA WORLD TOUR',
    e_id: 'G5vzZ4amDV3z0',
    url: 'https://www.altitudetickets.com/events/detail/pink-2019',
    img:
      'https://s1.ticketm.net/dam/a/1dd/d5e86d93-5e1a-49d9-b530-70fefc0f21dd_877061_RETINA_PORTRAIT_3_2.jpg',
    date: 'Mon, Apr 1, 2019 7:30 PM',
    venue_name: 'Pepsi Center',
    address: '1000 Chopper Circle Denver, CO 80204',
    lat: '39.74724',
    lng: '-105.010166',
    distance: 0.75,
    favorite: false
  },
  {
    name: 'Childish Gambino',
    e_id: 'G5vzZ4aV5O3CO',
    url: 'https://www.altitudetickets.com/events/detail/childish-gambino',
    img:
      'https://s1.ticketm.net/dam/a/7d9/44a96192-d246-4946-b6b4-c1f7f975a7d9_816871_TABLET_LANDSCAPE_3_2.jpg',
    date: 'Tue, Dec 4, 2018 7:30 PM',
    venue_name: 'Pepsi Center',
    address: '1000 Chopper Circle Denver, CO 80204',
    lat: '39.74724',
    lng: '-105.010166',
    distance: 0.75,
    favorite: false
  }
];

export const favoriteEventResponse = {
  message:
    'Successfully added EnParejaDos - con Adrian Uribe y Consuelo Duval with id: G5vzZ48H7Q7cH'
};
