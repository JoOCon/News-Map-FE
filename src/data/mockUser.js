export const mockUser = {
  El: '117977763637801404146',
  Zi: {
    token_type: 'Bearer',
    access_token:
      'ya29.GlxBBtt0Yi5X7awZrvhwo7UAQjrOLQ_JwGQ0ZVcJhahzVb4kj_Z5NqMx4wAhouEXxtNs72jp9pG6TZ68H1nf3bgknrUoiWzhRnotssJFcYMiopKmF1oc0ejm8ZDCwQ',
    scope:
      'openid email profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/plus.me https://www.googleapis.com/auth/userinfo.profile',
    login_hint:
      'AJDLj6JUa8yxXrhHdWRHIV0S13cArCX2skrQ7skgtwzDVxi66KXli5pV9QkfHh_W71S4j789mYsJSJTDHdBzZgGY2XexHHgGyA',
    expires_in: 3600,
    id_token:
      'eyJhbGciOiJSUzI1NiIsImtpZCI6IjcyOGY0MDE2NjUyMDc5YjllZDk5ODYxYmIwOWJhZmM1YTQ1YmFhODYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE3OTc3NzYzNjM3ODAxNDA0MTQ2IiwiZW1haWwiOiJweWtpbTA1OTFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiJNVS16RFc2c3lqbEhTNDZsbW5vMUVnIiwibmFtZSI6IlBhdWwgSyIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLXE3OWJ2NE5jX2RZL0FBQUFBQUFBQUFJL0FBQUFBQUFBQUFBL0FCdE5sYkRPMEIyU3A5U3pPWGVxZ0sxOWJvcXNUaUlWQUEvczk2LWMvcGhvdG8uanBnIiwiZ2l2ZW5fbmFtZSI6IlBhdWwiLCJmYW1pbHlfbmFtZSI6IksiLCJsb2NhbGUiOiJlbiIsImlhdCI6MTU0MDUwODUwMCwiZXhwIjoxNTQwNTEyMTAwLCJqdGkiOiI2ZTYyNTg0Mzc2NTYwNDMwNjA1Y2MxYmVlNTUxOTE5ZmNjNTVmN2VmIn0.HFfyTNUGKeLor17A-qWqSRNxfITUysVYoPwC2UFzbn06Ulx4GEj7NDm3qMXFQCxXO50dGvOcsF8MKvB2fsmRkdVZeoin5QCdZa43xbfxoTmUCPX-zKFNUt8Ln3rcewNuQubkZ15Tq1BvaIh-vOQLxGSi7_yZXhTnnGLz1QNYaIZkdMUILJkGlOqigCN68GgumUWePIWn73Z-OGeKV0AStyLWpXRYtO6gIv1o8H_XV1WT2Iud-sTINDKWSK9iuomoFEnAHRcxbOQDX9A9yHHv7ZxIyzzKTd1QQmxnB_Aiux0mxu_zsZiZU5J9LS24QV5_EaAgWVKj3nCLYytEOGbjJQ',
    session_state: { extraQueryParams: { authuser: '0' } },
    first_issued_at: 1540508499764,
    expires_at: 1540512099764,
    idpId: 'google'
  },
  w3: {
    Eea: '117977763637801404146',
    ig: 'Paul K',
    ofa: 'Paul',
    wea: 'K',
    Paa:
      'https://lh4.googleusercontent.com/-q79bv4Nc_dY/AAAAAAAAAAI/AAAAAAAAAAA/ABtNlbDO0B2Sp9SzOXeqgK19boqsTiIVAA/s96-c/photo.jpg',
    U3: 'pykim0591@gmail.com'
  },
  googleId: '117977763637801404146',
  tokenObj: {
    token_type: 'Bearer',
    access_token:
      'ya29.GlxBBtt0Yi5X7awZrvhwo7UAQjrOLQ_JwGQ0ZVcJhahzVb4kj_Z5NqMx4wAhouEXxtNs72jp9pG6TZ68H1nf3bgknrUoiWzhRnotssJFcYMiopKmF1oc0ejm8ZDCwQ',
    scope:
      'openid email profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/plus.me https://www.googleapis.com/auth/userinfo.profile',
    login_hint:
      'AJDLj6JUa8yxXrhHdWRHIV0S13cArCX2skrQ7skgtwzDVxi66KXli5pV9QkfHh_W71S4j789mYsJSJTDHdBzZgGY2XexHHgGyA',
    expires_in: 3600,
    id_token:
      'eyJhbGciOiJSUzI1NiIsImtpZCI6IjcyOGY0MDE2NjUyMDc5YjllZDk5ODYxYmIwOWJhZmM1YTQ1YmFhODYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE3OTc3NzYzNjM3ODAxNDA0MTQ2IiwiZW1haWwiOiJweWtpbTA1OTFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiJNVS16RFc2c3lqbEhTNDZsbW5vMUVnIiwibmFtZSI6IlBhdWwgSyIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLXE3OWJ2NE5jX2RZL0FBQUFBQUFBQUFJL0FBQUFBQUFBQUFBL0FCdE5sYkRPMEIyU3A5U3pPWGVxZ0sxOWJvcXNUaUlWQUEvczk2LWMvcGhvdG8uanBnIiwiZ2l2ZW5fbmFtZSI6IlBhdWwiLCJmYW1pbHlfbmFtZSI6IksiLCJsb2NhbGUiOiJlbiIsImlhdCI6MTU0MDUwODUwMCwiZXhwIjoxNTQwNTEyMTAwLCJqdGkiOiI2ZTYyNTg0Mzc2NTYwNDMwNjA1Y2MxYmVlNTUxOTE5ZmNjNTVmN2VmIn0.HFfyTNUGKeLor17A-qWqSRNxfITUysVYoPwC2UFzbn06Ulx4GEj7NDm3qMXFQCxXO50dGvOcsF8MKvB2fsmRkdVZeoin5QCdZa43xbfxoTmUCPX-zKFNUt8Ln3rcewNuQubkZ15Tq1BvaIh-vOQLxGSi7_yZXhTnnGLz1QNYaIZkdMUILJkGlOqigCN68GgumUWePIWn73Z-OGeKV0AStyLWpXRYtO6gIv1o8H_XV1WT2Iud-sTINDKWSK9iuomoFEnAHRcxbOQDX9A9yHHv7ZxIyzzKTd1QQmxnB_Aiux0mxu_zsZiZU5J9LS24QV5_EaAgWVKj3nCLYytEOGbjJQ',
    session_state: { extraQueryParams: { authuser: '0' } },
    first_issued_at: 1540508499764,
    expires_at: 1540512099764,
    idpId: 'google'
  },
  tokenId:
    'eyJhbGciOiJSUzI1NiIsImtpZCI6IjcyOGY0MDE2NjUyMDc5YjllZDk5ODYxYmIwOWJhZmM1YTQ1YmFhODYiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiMjQxNzc5OTQ0MDAyLXFhNDJ2NmN2azNrcHU4ZGEwamltZHIzNjR2bjQxdDFyLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTE3OTc3NzYzNjM3ODAxNDA0MTQ2IiwiZW1haWwiOiJweWtpbTA1OTFAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImF0X2hhc2giOiJNVS16RFc2c3lqbEhTNDZsbW5vMUVnIiwibmFtZSI6IlBhdWwgSyIsInBpY3R1cmUiOiJodHRwczovL2xoNC5nb29nbGV1c2VyY29udGVudC5jb20vLXE3OWJ2NE5jX2RZL0FBQUFBQUFBQUFJL0FBQUFBQUFBQUFBL0FCdE5sYkRPMEIyU3A5U3pPWGVxZ0sxOWJvcXNUaUlWQUEvczk2LWMvcGhvdG8uanBnIiwiZ2l2ZW5fbmFtZSI6IlBhdWwiLCJmYW1pbHlfbmFtZSI6IksiLCJsb2NhbGUiOiJlbiIsImlhdCI6MTU0MDUwODUwMCwiZXhwIjoxNTQwNTEyMTAwLCJqdGkiOiI2ZTYyNTg0Mzc2NTYwNDMwNjA1Y2MxYmVlNTUxOTE5ZmNjNTVmN2VmIn0.HFfyTNUGKeLor17A-qWqSRNxfITUysVYoPwC2UFzbn06Ulx4GEj7NDm3qMXFQCxXO50dGvOcsF8MKvB2fsmRkdVZeoin5QCdZa43xbfxoTmUCPX-zKFNUt8Ln3rcewNuQubkZ15Tq1BvaIh-vOQLxGSi7_yZXhTnnGLz1QNYaIZkdMUILJkGlOqigCN68GgumUWePIWn73Z-OGeKV0AStyLWpXRYtO6gIv1o8H_XV1WT2Iud-sTINDKWSK9iuomoFEnAHRcxbOQDX9A9yHHv7ZxIyzzKTd1QQmxnB_Aiux0mxu_zsZiZU5J9LS24QV5_EaAgWVKj3nCLYytEOGbjJQ',
  accessToken:
    'ya29.GlxBBtt0Yi5X7awZrvhwo7UAQjrOLQ_JwGQ0ZVcJhahzVb4kj_Z5NqMx4wAhouEXxtNs72jp9pG6TZ68H1nf3bgknrUoiWzhRnotssJFcYMiopKmF1oc0ejm8ZDCwQ',
  profileObj: {
    googleId: '117977763637801404146',
    imageUrl:
      'https://lh4.googleusercontent.com/-q79bv4Nc_dY/AAAAAAAAAAI/AAAAAAAAAAA/ABtNlbDO0B2Sp9SzOXeqgK19boqsTiIVAA/s96-c/photo.jpg',
    email: 'pykim0591@gmail.com',
    name: 'Paul K',
    givenName: 'Paul',
    familyName: 'K'
  }
};

export const mockStateUser = {
  first_name: 'Paul',
  last_name: 'K',
  email: 'pykim0591@gmail.com',
  gid: '117977763637801404146',
  id: 3,
  created_at: '2018-10-24T23:37:44.724Z',
  updated_at: '2018-10-24T23:37:44.724Z'
};

export const mockUserResponse = {
  created_at: '2018-10-24T23:37:44.724Z',
  email: 'pykim0591@gmail.com',
  first_name: 'Paul',
  gid: '117977763637801404146',
  id: 3,
  last_name: 'K',
  updated_at: '2018-10-24T23:37:44.724Z'
};
