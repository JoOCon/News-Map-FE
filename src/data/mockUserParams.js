export const responseGoogle1 =
  'https://event-mapper-api.herokuapp.com/api/v1/users';

export const responseGoogle2 = {
  body:
    '{"user":{"google_id":"117977763637801404146","given_name":"Paul","family_name":"K","email":"pykim0591@gmail.com"}}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const getEvents =
  'https://app.ticketmaster.com/discovery/v2/events.json?apikey=GddBLHQGZLJjPNWsNxOrYRAoBx8Hdqrf&geoPoint=9wjr4et3f&radius=200&size=100';

export const postUser = {
  body: '{"user":{"email":"pykim0591@gmail.com"}}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const setFavorite = {
  body: '{}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const removeEvent =
  'https://event-mapper-api.herokuapp.com/api/v1/users/undefined/events/undefined';

export const removeEventOptions = {
  headers: { 'Content-Type': 'application/json' },
  method: 'DELETE'
};
