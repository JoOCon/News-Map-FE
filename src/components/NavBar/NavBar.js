import React, { Component } from 'react';
import { connect } from 'react-redux';
import { Route, Link } from 'react-router-dom';
import PropTypes from 'prop-types';
import './NavBar.css';

export class NavBar extends Component {
  render() {
    const { gid, first_name } = this.props.activeUser;
    const loggedIn = () => {
      return (
        <div className='nav-container'>
          <h1>{`Hello, ${first_name}`}</h1>
          <div className='button-container'>
            <Link to={`/app/${gid}/profile`}>
              <button className='main-button'>Your Profile</button>
            </Link>
            <Link to={`/app/${gid}/favorites`}>
              <button className='main-button'>Favorites</button>
            </Link>
          </div>
        </div>
      );
    };

    const favorites = () => {
      return (
        <div className='nav-container'>
          <div className='button-container'>
            <Link to={`/app/${gid}/profile`}>
              <button className='main-button'>Your Profile</button>
            </Link>
            <Link to={`/app`}>
              <button className='main-button'>Current Events</button>
            </Link>
          </div>
        </div>
      );
    };

    const profile = () => {
      return (
        <div className='nav-container'>
          <Link to={`/app/${gid}/favorites`}>
            <button className='main-button'>Favorites</button>
          </Link>
          <Link to={`/app`}>
            <button className='main-button'>Current Events</button>
          </Link>
        </div>
      );
    };

    return (
      <div className='main-nav'>
        <Route exact path={`/app`} component={loggedIn}></Route>
        <Route exact path={`/app/${gid}/favorites`} component={favorites}></Route>
        <Route exact path={`/app/${gid}/profile`} component={profile}></Route>
      </div>
    );
  }
}

export const mapStateToProps = (state) => ({
  activeUser: state.activeUser
});

NavBar.propTypes = {
  activeUser: PropTypes.object
};

export default connect(mapStateToProps)(NavBar);
