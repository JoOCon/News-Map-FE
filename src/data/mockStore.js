import * as user from './mockUser';
import * as events from './mockEvents';

export const mockStore = {
  activeUser: user.mockStateUser,
  userLocation: { latitude: 39.43243, longitude: -104.98434 },
  watchList: events.userWatchlist,
  events: events.eventsResponse
};
