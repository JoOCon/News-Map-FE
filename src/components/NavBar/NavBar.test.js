<<<<<<< HEAD
import React from 'react';

import * as user from '../../data/mockUser';
import { NavBar } from './NavBar';

describe('NavBar', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NavBar activeUser={user.mockStateUser} />);
  });

  it('should matchSnapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
=======
// import React from 'react';
// import NavBar from './NavBar';

it('renders without crashing', () => {
  expect(false).toEqual(false);
});
>>>>>>> bb0d2f5133f99f48d80daa814d97726ebfb28c6c
