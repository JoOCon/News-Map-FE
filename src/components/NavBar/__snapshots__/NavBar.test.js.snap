// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should matchSnapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar
    activeUser={
      Object {
        "created_at": "2018-10-24T23:37:44.724Z",
        "email": "pykim0591@gmail.com",
        "first_name": "Paul",
        "gid": "117977763637801404146",
        "id": 3,
        "last_name": "K",
        "updated_at": "2018-10-24T23:37:44.724Z",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="greeting"
        >
          Hello, Paul
        </h1>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link"
          exact={true}
          to="/"
        >
          Home
        </NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link"
          to="/117977763637801404146/profile"
        >
          Profile
        </NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link"
          to="/117977763637801404146/favorites"
        >
          Favorites
        </NavLink>,
      ],
      "className": "nav-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hello, Paul",
          "className": "greeting",
        },
        "ref": null,
        "rendered": "Hello, Paul",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "Home",
          "className": "nav-link",
          "exact": true,
          "to": "/",
        },
        "ref": null,
        "rendered": "Home",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "Profile",
          "className": "nav-link",
          "to": "/117977763637801404146/profile",
        },
        "ref": null,
        "rendered": "Profile",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": "Favorites",
          "className": "nav-link",
          "to": "/117977763637801404146/favorites",
        },
        "ref": null,
        "rendered": "Favorites",
        "type": [Function],
      },
    ],
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="greeting"
          >
            Hello, Paul
          </h1>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link"
            exact={true}
            to="/"
          >
            Home
          </NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link"
            to="/117977763637801404146/profile"
          >
            Profile
          </NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link"
            to="/117977763637801404146/favorites"
          >
            Favorites
          </NavLink>,
        ],
        "className": "nav-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hello, Paul",
            "className": "greeting",
          },
          "ref": null,
          "rendered": "Hello, Paul",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Home",
            "className": "nav-link",
            "exact": true,
            "to": "/",
          },
          "ref": null,
          "rendered": "Home",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Profile",
            "className": "nav-link",
            "to": "/117977763637801404146/profile",
          },
          "ref": null,
          "rendered": "Profile",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Favorites",
            "className": "nav-link",
            "to": "/117977763637801404146/favorites",
          },
          "ref": null,
          "rendered": "Favorites",
          "type": [Function],
        },
      ],
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
